.experimenter.yaml

```diff
@@ -33,8 +33,8 @@ messaging:
       type: string
       description: What should be displayed when a control message is selected.
       enum:
-        - show-next-message
         - show-none
+        - show-next-message
```

Comment:
@jhugman Do you have any idea why this change keeps being made on app build? Should I commit this change to the main branch (on a separate commit/ PR), or is it something local and I should ignore it? Thank you!